instrumentation/que/lib/opentelemetry/instrumentation/que/patches/que_job.rb

```diff
@@ -46,7 +46,8 @@ def enqueue(*args, job_options: {}, **arg_opts)
                 # In Que version 2.1.0 `bulk_enqueue` was introduced and in order
                 # for it to work, we must pass `job_options` to `bulk_enqueue` instead of enqueue.
                 if gem_version >= Gem::Version.new('2.1.0') && Thread.current[:que_jobs_to_bulk_insert]
-                  Thread.current[:que_jobs_to_bulk_insert][:job_options] = job_options.merge(tags: tags)
+                  Thread.current[:que_jobs_to_bulk_insert][:job_options] = bulk_job_options(tags)
```

Comment:
Similarly, could you issue dig with safe navigation? I also think it would be ok to in-line the helper method.

Despite our gratitude and desire for clean code, the less methods we dispatch to will result in better overall instrumentation performance and the smaller the stack trace when applications raise exceptions.